server/delay_server/src/irodsDelayServer.cpp

```diff
@@ -666,7 +666,7 @@ int main(int argc, char** argv)
 
```

Comment:
I can't comment up there, but I see some "Rule execution server" on 621, 654, and 659.

Reply:
Will change.

Reply:
and 653

Reply:
done